.devcontainer/docker-compose.yml

```diff
@@ -1,37 +1,85 @@
+# Defines the services and workspace environment used for codespaces
+#
+# Used
+# https://github.com/microsoft/vscode-dev-containers/blob/main/containers/python-3-postgres/.devcontainer/docker-compose.yml
+# for reference.
+
 version: '3'
 services:
-    # Update this to the name of the service you want to work with in your docker-compose.yml file
-    web:
-        # If you want add a non-root user to your Dockerfile, you can use the "remoteUser"
-        # property in devcontainer.json to cause VS Code its sub-processes (terminals, tasks,
-        # debugging) to execute as the user. Uncomment the next line if you want the entire
-        # container to run as this user instead. Note that, on Linux, you may need to
-        # ensure the UID and GID of the container user you create matches your local user.
-        # See https://aka.ms/vscode-remote/containers/non-root for details.
-        #
-        # user: vscode
-
-        # Uncomment if you want to override the service's Dockerfile to one in the .devcontainer
-        # folder. Note that the path of the Dockerfile and context is relative to the *primary*
-        # docker-compose.yml file (the first in the devcontainer.json "dockerComposeFile"
-        # array). The sample below assumes your primary file is in the root of your project.
-        #
-        # build:
-        #   context: .
-        #   dockerfile: .devcontainer/Dockerfile
-
+    # The workspace environment you'll be dropped into. Note that I do not
+    # create separate services for each process e.g. celery, the plugin server.
+    # I expect these to be run from within this workspace context, such that you
+    # do not need to interact with them via docker, the intent is to make them
+    # readily debuggable.
+    #
+    # Service dependencies are, however, run separately as there isn't so much
+    # of a need to debug them. You can of course however use `docker logs` (as we
+    # have docker-in-docker enabled) to view these, of via the vscode docker
+    # extention.
+    app:
+        image: ghcr.io/posthog/posthog/codespaces:master
```

Comment:
why not pushing directly to DockerHub (like we do with other images)?

Reply:
These images I don't think are relevant outside of github, and github has tooling around ghcr, so thought I'd lean into github a bit for this. Do you think worth changing?